### PR TITLE
[FEATURE] install/configure filebeat and metricbeat

### PR DESCRIPTION
**Pull request by dipiazzg**

### CODE REVIEW

File: config/templates/etc/filebeat/filebeat.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+# Below are the input specific configurations.
+
+- type: log
+
+  # Change to true to enable this input configuration.
+  enabled: true
+
+  # Paths that should be crawled and fetched. Glob based paths.
+  paths:
+    - /var/log/*.log
```
Comment (eolix) [inline config/templates/etc/filebeat/filebeat.yml.j2]: I would add `- /var/log/messages` and `-  /var/log/dmesg` to cover for the RHEL-based distros as well.

Comment (dipiazzg) [reply config/templates/etc/filebeat/filebeat.yml.j2:28]: This is now fixed accordingly

**Comment by dipiazzg** [comment]
> Not a deal-breaker, but should `logstash_endpoint` be something like `beats_target_hosts`, since logstash is only one type recipient of the beats logs (might use fluentd)?
> 
> Can I also suggest that you allow it to be defined as an array (currently you are limiting yourself to a single target) and add an example (uninitialised) definition to `EXAMPLE/group_vars/_skel/cluster_vars.yml`.
> 
> `beats_target_hosts: [] # The destination hosts for e.g. filebeat/ metricbeat logs`

Thanks for the hints, I've now amended those.

**Comment by dipiazzg** [comment]
Fixed (thanks again for the hint)

**Pull request merged**